src/sql/migrations.rs

```diff
@@ -497,6 +497,18 @@ item TEXT DEFAULT '');"#,
         )
         .await?;
     }
+    if dbversion < 81 {
+        info!(context, "[migration] v81");
+        sql.execute_migration(
+            r#"CREATE TABLE w30_status_updates (
+id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
why is `AUTOINCREMENT` required?

Reply:
iirc, this ensures a once used id is not used again, which looks like a good thing to have in general when passing ids around (we are using that for all other tables) - otherwise, eg. an entry may get deleted and a new entry will get the same id. however, maybe i also misunderstood AUTOINCREMENT

Reply:
yes, you remember correctly.  but not all tables use it and sqlite recommends to not use it if the table doesn't need it (it is expensive). so this is only needed if:
- you will delete rows
- you might still perform a lookup by id after a row is potentially deleted

Reply:
> you might still perform a lookup by id after a row is potentially deleted

maybe it is not needed, however, it is good to have this second defence. iirc, we had issues with that way back when some thread is pending around, with some message-id/images that is deleted meanwhile - and then surprisingly get a different content.

for the costs: stackoverflow says sth. about 8% or so, probably on inserts only, so i suspect that this is the bottleneck somewhere in our stack. if so, and we are sure there are no races or so, we can remove it at some point, that is easier doable than reacting so some weird bugs we may oversee during developlement.

Reply:
quite possible it is the bottleneck I perceived when sending an status-update in a big account/db (sending status from javascript api took like 1-2 seconds) agree it can be left for the future, but would be good the javascript api do the call in another thread and returns immediately otherwise the app gets frozen for some seconds

Reply:
> quite possible it is the bottleneck I perceived when sending an status-update in a big account/db (sending status from javascript api took like 1-2 seconds) agree it can be left for the future, but would be good the javascript api do the call in another thread and returns immediately otherwise the app gets frozen for some seconds

AUTOINCREMENT or not, `dc_send_w30_status_update()` should return within very few ms, if not, this is a bug. a simple insert must not take 1-2 seconds - and the sending of the message is done as usual in the background, using the same system as for normal messages.

of course, receiving the message may take 1-2 seconds - or even more.

so, before taking any action on that, we should measure, what really consumes the 1-2 seconds - if that is really that slow at the end, not sure what exactly the "fake implementation" is doing.

Reply:
i think send message is or was the problem, it's sync and as such might freeze the ui

Reply:
yip, let's resolve this here, and open an issue as needed. it get's too confusing.